src/Orchard.Web/Modules/Orchard.ImportExport/Services/SetupService.cs

```diff
@@ -101,6 +102,10 @@ public class SetupService : Component, ISetupService {
                     schemaBuilder.AlterTable("Orchard_Framework_DataMigrationRecord",
                         table => table.AddUniqueConstraint("UC_DMR_DataMigrationClass_Version", "DataMigrationClass", "Version"));
 
+                    // Create the distributed lock record schema.
```

Comment:
I insist! This table is created in the automatic migration event handler. And the setup will also call the automatic migration event handler, so you don't need to call the schema migration from the setup services. Or is it not called during setups?